src/Sarif/AnalyzeContextBase.cs

```diff
@@ -34,7 +34,7 @@ public IEnumerable<IOption> GetOptions()
         public abstract IAnalysisLogger Logger { get; set; }
         public abstract RuntimeConditions RuntimeErrors { get; set; }
         public abstract bool AnalysisComplete { get; set; }
-        public abstract DefaultTraces Traces { get; set; }
```

Comment:
>DefaultTraces [](http://example.com/codeflow?start=24&length=13)

The issue was in sdk it is trying to convert PdbLoad into DefaultTraces enum and it will throw exception.
so I changed it and all the implementation of it to avoid this issue,